velox/functions/sparksql/Arithmetic.h

```diff
@@ -262,6 +262,16 @@ struct Log1pFunction {
   }
 };
 
+
+struct Expm1Function {
```

Comment:
@Donvi could you add the template parameter like in the other functions? This is breaking one of our internal builds.

Reply:
Thanks, Done．